extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -388,6 +388,7 @@ describe('amp-a4a', () => {
         const a4aElement = createA4aElement(doc);
         a4aElement.setAttribute('type', 'adsense');
         const a4a = new AmpA4A(a4aElement);
+        a4a.buildCallback();
```

Comment:
this is something I don't understand. I assume runtime is on by default when we create test iframe. However I have to manually call `#buildCallback` here to get the correct `this.config` setting.